src/sections/Community/Handbook/faq.js

```diff
@@ -17,43 +17,43 @@ const Faq = () => {
             <h2>General FAQs:</h2><br />
-            <h4>Q: Are Layer5’s solutions open source?</h4>
-            <p>A: Yes, all Layer5 projects are licensed under Apache V2. Layer5 also offers extensions to the Meshery project in which Enterprise-centric functionality is commercially offered and supported.</p>
+            <h3>Q: Are <strong>Layer5’s</strong> solutions open source?</h3>
+            <p><strong>A:</strong> Yes, all <strong>Layer5</strong> projects are licensed under Apache V2. <strong>Layer5</strong> also offers extensions to the Meshery project in which Enterprise-centric functionality is commercially offered and supported.</p>
 
-            <h4>Q: I am new to Open Source, where do I begin?</h4>
-            <p>A: You can start by going through the <a href="https://layer5.io/community/newcomers">Newcomers’ Welcome Guide.</a></p>
+            <h3>Q: I am new to Open Source, where do I begin?</h3>
+            <p><strong>A:</strong> You can start by going through the <a href="https://layer5.io/community/newcomers">Newcomers’ Welcome Guide.</a></p>
 
-            <h4>Q: What exactly is a MeshMate?</h4>
-            <p>A:Layer5<Link to="/community/meshmates"> MeshMates </Link>are individuals committed to helping community members be successful contributors. Meshmates are there to guide and support the community members, helping them identify projects they can contribute to depending on their areas of interest, directing them on groups to join and helping them grow in their open-source and cloud native knowledge.</p>
+            <h3>Q: What exactly is a MeshMate?</h3>
+            <p><strong>A:</strong> <strong>Layer5</strong><Link to="/community/meshmates"> MeshMates </Link>are individuals committed to helping community members be successful contributors. Meshmates are there to guide and support the community members, helping them identify projects they can contribute to depending on their areas of interest, directing them on groups to join and helping them grow in their open-source and cloud native knowledge.</p>
 
-            <h4>Q: How do I join the mailing list?</h4>
-            <p>A: You can subscribe to the Layer5 mailing <a href="https://layer5.io/subscribe">here.</a></p>
+            <h3>Q: How do I join the mailing list?</h3>
+            <p><strong>A:</strong> You can subscribe to the <strong>Layer5</strong> mailing <a href="https://layer5.io/subscribe">here.</a></p>
 
-            <h4>Q: How do I get an internship in Layer5?</h4>
-            <p>A: To best position your candidacy for an internship with Layer5, engage in the community and its projects. Start contributing and keep contributing. Community members who consistently contribute are the first individuals to be awarded internships when new internship opportunities open up - particularly members who are making significantly impactful contributions.</p>
+            <h3>Q: How do I get an internship in <strong>Layer5</strong>?</h3>
```

Comment:
While you are on this page , you can change it to " How do I get an internship at Layer5?"

Reply:
Okay Sure , anything else you want me add or change ?